renderdoc/driver/vulkan/wrappers/vk_shader_funcs.cpp

```diff
@@ -89,6 +89,23 @@ static RDResult DeferredPipelineCompile(VkDevice device,
   VkRayTracingPipelineCreateInfoKHR *unwrapped =
       UnwrapStructAndChain(CaptureState::LoadingReplaying, mem, &createInfo);
 
+  // Patch in capture/replay creation flags
```

Comment:
We should do this the same way as we handle capture/replay bits for BDA: Patch it into the serialised form during capture rather than doing it independently on replay.